client/java/src/main/java/com/linecorp/centraldogma/client/CentralDogma.java

```diff
@@ -452,10 +456,36 @@
      * @return the latest known {@link Revision} which contains the changes for the matched files.
      *         {@code null} if the files were not changed for {@code timeoutMillis} milliseconds
      *         since the invocation of this method.
+     *
+     * @deprecated Use {@link CentralDogma#watchRepository(String, String, Revision, String, long, boolean)}.
+     */
+    @Deprecated
+    default CompletableFuture<Revision> watchRepository(String projectName, String repositoryName,
+                                                        Revision lastKnownRevision, String pathPattern,
+                                                        long timeoutMillis) {
+        return watchRepository(projectName, repositoryName, lastKnownRevision, pathPattern,
+                               timeoutMillis, WatchConstants.DEFAULT_WATCH_ERROR_ON_ENTRY_NOT_FOUND);
+    }
+
+    /**
+     * Waits for the files matched by the specified {@code pathPattern} to be changed since the specified
+     * {@code lastKnownRevision}. If the files don't exist and {@code errorOnEntryNotFound} is ture, stops to
+     * wait files and throw {@link EntryNotFoundException} as the cause. If {@code errorOnEntryNotFound} is
+     * false and no changes were made within the specified {@code timeoutMillis}, the returned {@link
+     * CompletableFuture} will be completed with {@code null}. It is recommended to specify the largest {@code
+     * timeoutMillis} allowed by the server.
+     *
+     * <p>Note: Legacy client does not support {@code errorOnEntryNotFound}</p>
+     *
+     * @return the {@link Entry} which contains the latest known {@link Query} result.
+     *         thrown {@link EntryNotFoundException} if files don't exist and
+     *         {@code errorOnEntryNotFound} is ture.
+     *         {@code null} if the file was not changed for {@code timeoutMillis} milliseconds
+     *         since the invocation of this method.
      */
```

Comment:
I'm not sure this is the best but how about:
```java
    /**
     * Waits for the files matched by the specified {@code pathPattern} to be changed since the specified
     * {@code lastKnownRevision}. If the files don't exist and {@code errorOnEntryNotFound} is {@code true},
     * the returned {@link CompletableFuture} will be completed exceptionally with
     * {@link EntryNotFoundException}. If no changes were made within the specified {@code timeoutMillis},
     * the returned {@link CompletableFuture} will be completed with {@code null}.
     * It is recommended to specify the largest {@code timeoutMillis} allowed by the server.
     *
     * <p>Note: Legacy client does not support {@code errorOnEntryNotFound}</p>
     *
     * @return the latest known {@link Revision} which contains the changes for the matched files.
     *         {@code null} if the files were not changed for {@code timeoutMillis} milliseconds
     *         since the invocation of this method. {@link EntryNotFoundException} is raised if the
     *         target does not exist.
     */

```

Reply:
Thanks. :)
I made a mistake in copy and paste.